org.zaproxy.ZAP.json

```diff
@@ -43,7 +45,14 @@
                 {
                     "type": "archive",
                     "url": "https://github.com/zaproxy/zaproxy/releases/download/v2.13.0/ZAP_2.13.0_Linux.tar.gz",
-                    "sha256": "936eb52a0fd390c1ef890c455420d95ce20062fe136ec0927e023e2baf50f549"
+                    "sha256": "936eb52a0fd390c1ef890c455420d95ce20062fe136ec0927e023e2baf50f549",
+                    "x-checker-data": {
```

Comment:
Revert this as well, we have our own release workflow which will take care of this.

Reply:
Reverted, :)